pyis/share/str_utils.cpp

```diff
@@ -119,12 +121,218 @@ char32_t StripAccent(char32_t c) {
     return tr[c - 192];
 }
 
-void FindAndReplaceAll(std::string& data, const std::string& str_to_search, const std::string& str_to_replace) {
+void replace_all(std::string& data, const std::string& str_to_search, const std::string& str_to_replace) {
     size_t pos = data.find(str_to_search);
     while (pos != std::string::npos) {
         data.replace(pos, str_to_search.length(), str_to_replace);
         pos = data.find(str_to_search, pos + str_to_replace.length());
     }
 }
 
-}  // namespace pyis
+bool is_unicode_category_L(const char32_t& ch) {
```

Comment:
We could rename those two function to a more understandable name, such as `is_letter` & `is_number`.